conftest.py

```
@@ -491,20 +491,31 @@ def pytest_report_teststatus(report, config):
     test_name = report.head_line
     when = report.when
     call_str = "call"
+
     if report.passed:
         if when == call_str:
             BASIC_LOGGER.info(f"\nTEST: {test_name} STATUS: \033[0;32mPASSED\033[0m")
+        return None
 
     elif report.skipped:
-        BASIC_LOGGER.info(f"\nTEST: {test_name} STATUS: \033[1;33mSKIPPED\033[0m")
+        type = "XFAILED" if report.wasxfail else "SKIPPED"
```

Pylint (3.3.7)

[warning] Redefining built-in 'type'

(W0622)